system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -161,6 +161,8 @@ registry:
             item: "https://staging.dchealthlink.com/individuals/tax-documents"
           - key: :statewide_area
             item: "DC-Metro"
+          - key: :nationwide_area
```

Comment:
what is the purpose of this setting? does it change between clients?

Reply:
network information related to DC plans. its specific to DC only.